utilities/qneg.F90

```diff
@@ -0,0 +1,388 @@
+module qneg
+
```

Comment:
It appears that this module may just be for CCPP use (and not being taken back into ESCOMP/CAM).  If that is the case, I suggest that a comment be added at the top of this module indicating its limited scope.

Reply:
added a comment.